docs/contributing/documentation.md

```diff
@@ -6,40 +6,30 @@
 
 ## Develop
 
+1. Make changes on a development branch
+
 1. Run the following command to render and serve documentation locally
 
         docker-compose run --rm --service-ports mkdocs
 
 1. Visit the local documentation to see your changes while developing  
     **[http://localhost:8000](http://localhost:8000)**  
+  
+1. Push this branch, get approval and merge to the `main` branch
 
 ## Publish
 
 1. Pull down the `gh-pages` branch. This only needs to be done once
 
         git checkout --track origin/gh-pages
     
-1. Generate documentation on your development branch
+1. Pull down the `main` branch after changes have been merged
```

Comment:
Do we need this step? The script does all of those things already. I would delete to reduce confusion.

Reply:
good catch, thanks!